samples/sampler/src/MoonstoneEnvironment/MoonstoneEnvironment.js

```diff
@@ -36,20 +36,28 @@ const PanelsBase = kind({
 		className: 'moonstoneEnvironmentPanels'
 	},
 
-	render: ({children, description, noHeader, noPanel, noPanels, title, ...rest}) => (
-		!noPanels ? <Panels {...rest} onApplicationClose={reloadPage}>
-			{!noPanel ? <Panel className={css.panel}>
-				{!noHeader ? [<Header type="compact" title={title} key="header" />,
-					<Column key="body">
-						{description ? (
-							<Cell shrink component={BodyText} className={css.description}>{description}</Cell>
-						) : null}
-						<Cell className={css.storyCell}>{children}</Cell>
-					</Column>] : children
-				}
-			</Panel> : children}
-		</Panels> : <div {...rest}>{children}</div>
-	)
+	render: ({children, description, noHeader, noPanel, noPanels, title, className, ...rest}) => {
```

Comment:
Please use the Enact platform module https://github.com/enactjs/moonstone/pull/92/files to get the android platform information

Reply:
I moved the solution from the sampler to Panels and used Enact platform to check for the platform.

Reply:
@alexandrumorariu Did you check the link that I shared? https://stackoverflow.com/questions/23757345/android-does-not-correctly-scroll-on-input-focus-if-not-body-element
I would like not to apply the extra margin in the panel.